### ISSUE TITLE
SourceForge sample set list out of date, misleading, not so useful

### ISSUE THREAD

**Issue by dgatwood** [GrandOrgue/grandorgue#1968]
I was really excited to try out this software, but I've been completely unable to find any usable free or demo organ sample sets using your list on SourceForge.  In particular:

1.  A large percentage of the links are dead:

- Nicholas Appleton per-page links (the download links in the referenced post do still work).
- Prudhoe Methodist Church
- Virtual-Organs.nl successor
- South African Virtual Pipe Organ Project
- Prospectum Happy Birthday organ
- https://plus.google.com/103958129553708971826/posts
- MPS Orgelseite  (https://grandorgue.de)
- Maltese Historic Pipe Organs
- Piotr Grabowski
- Panos Virtual Organs
- BONZI ANDREA ANGELO

It really doesn't make sense to list things that are no longer available.  And removing them wipes out a rather large percentage of the available sample libraries, so this is a little disconcerting.  GrandOrgue would do well to provide hosting for sample libraries so that availability does not degrade any further.

2.  It's worth explicitly pointing out that the Lars Palo, Walcker Wildervank Complete, Nicholas Appleton, and Pere Casulleras sample sets are ALL under one form or another of Creative Commons licenses.

3.  The Jean-Luc Derouineau organ has no licensing terms that I can find.  If you have contact info, any chance you could convince him to add a LICENSE file?

4.  Ott Orgel appears to be a Hauptwerk sample set, not a GrandOrgue set (unless there's a GrandOrgue ODL file hiding in the installation archive somewhere).

Unfortunately, the organ I'm setting up will be in a public rehearsal space, and I have no way to force users to agree to licensing terms before they use it.  As a result, I cannot safely install any sample library that is CC-licensed unless it is either CC0 or provides an explicit, completely restriction-free license grant for any downstream recordings made with the software, which unfortunately, exactly none of the free CC-licensed sample libraries do.

Without such a grant, one of my random coworkers could:

- unknowingly use the sample libraries to create music for advertising their book.
- unknowingly use it in a recording that they sell.
- unknowingly distribute their derivative recording under a non-CC license.
- upload a derivative recording to a site like YouTube or Facebook, which, by virtue of preventing downloads, prevents downstream users from exercising their rights, potentially causing the uploader to violate ShareAlike licensing terms.

And this could all happen without my knowledge.  These and many other circumstances could get both the organ user and me into serious trouble.  To be fair, I suspect most people don't think about those issues when they choose a license, and that many of them did not intend for incidental use of organ sample libraries to taint everything that they touch, but I don't have the luxury of hoping that they don't care enough to sue me or some random coworker that I've never met.

The bottom line is that although I found harpsichord, piano, and carillon sample libraries, I found exactly *zero* free organ sample libraries with GrandOrgue ODLs that aren't CC-license-encumbered.

Am I missing something?  Is buying Hauptwerk really the only option that won't open me up to potentially infinite legal liability?

It seems like there's a lot of room for improving documentation, sample library distribution, etc., and for encouraging sample library creators to use more permissive licenses that still prevent people from stealing their work to use in sample libraries, but without arbitrarily limiting downstream use in created mixes.

**Comment by dgatwood** [comment]
Correction: There was precisely one suitably licensed organ: orgue de salon.  Sorry I missed that.

**Comment by rousseldenis** [comment]
@dgatwood Thank you for your report. But as said on Sourceforge website(you found it apparently), Gîthub is our new home.

The links for sample sets are here: https://github.com/GrandOrgue/grandorgue/wiki/Sample-Sets

Unfortunately, about licensing, we only manage GrndOrgue one. Sample sets producers are responsible and have the full choice under which license they publish them.

I would say, and you are right on that point, people usually explain on which manner their sample sets should be used before download step but a lot do not provide explicitly the license for them.

This is not an issue of GrandOrgue software and should be moved to discussions.

**Comment by larspalo** [comment]
> Am I missing something?

I think you're slightly mistaken about how the CC BY-SA license that I use for my sample sets would apply.

The license per se covers and carries over to derivative works of the sample set itself. This could for instance be a HW port of the sample set, a soundfont derivative using the samples for jOrgan or similar. The license also stay with the sample set even in an organ console that's using the samples embedded. Using the samples or just re-recording the samples for usage in any type of instrument what so ever would also of course be covered.

But, and this is important, a recording of someone playing a musical piece using a CC BY-SA licensed sample set is not at all covered or "encumbered" by the CC BY-SA license. The performance as such is their own to do whatever they want as long as it reaches a level of distinction higher than just playing the "samples" with the intention to create a derivative work...

In fact, there's nothing in the CC BY-SA that even prevents commercial usage of the sample set! I personally don't charge for them but it would be perfectly acceptable, from a license point of view, that someone else did take money to re-distribute them! One case might be an organ builder that makes a console for someone else using the samples - it's nothing wrong with paying the organ builder for it! The sample set as such is always covered by the license though so if the buyer decides to share it, that's also perfectly fine and cannot be prohibited by the organ builder.

More information can be found at https://creativecommons.org/licenses/by-sa/4.0/ which is kind of what I use for all new sample sets I produce.

However, on a side note, I've noticed that even the list of sample sets we have here at Github contains a few links that doesn't work that either should be fixed or removed.

**Comment by oleg68** [comment]
Sourceforge web page is not more maintained
